airbyte-webapp/src/packages/cloud/views/layout/SideBar/SideBar.tsx

```diff
@@ -1,191 +1,180 @@
 import SourceIcon from "views/layout/SideBar/components/SourceIcon";
 import { NotificationIndicator } from "views/layout/SideBar/NotificationIndicator";
-import { useCalculateSidebarStyles, getPopoutStyles } from "views/layout/SideBar/SideBar";
+import { useCalculateSidebarStyles } from "views/layout/SideBar/SideBar";
 
+import { useConfig } from "../../../../../config";
 import { RoutePaths } from "../../../../../pages/routePaths";
-
-const Bar = styled.nav`
-  width: 100px;
-  min-width: 65px;
-  height: 100%;
-  background: ${({ theme }) => theme.darkPrimaryColor};
-  padding: 23px 3px 15px 4px;
-  text-align: center;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  position: relative;
-  z-index: 9999;
-`;
-
-const Menu = styled.ul`
-  padding: 0;
-  margin: 20px 0 0;
-  width: 100%;
-`;
-
-const Text = styled.div`
-  margin-top: 7px;
-`;
-
-const WorkspaceButton = styled.div`
-  font-size: 9px;
-  line-height: 21px;
-  font-weight: 400;
-  height: 21px;
-  color: ${({ theme }) => theme.whiteColor};
-  border-radius: 10px;
-  margin-top: 13px;
-  background: rgba(255, 255, 255, 0.2);
-  cursor: pointer;
-  display: block;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
-  padding: 0 8px;
-  text-align: center;
-`;
+import ChatIcon from "../../../../../views/layout/SideBar/components/ChatIcon";
+import DocsIcon from "../../../../../views/layout/SideBar/components/DocsIcon";
+import RecipesIcon from "../../../../../views/layout/SideBar/components/RecipesIcon";
+import {
+  SidebarDropdownMenu,
+  SidebarDropdownMenuItemType,
+} from "../../../../../views/layout/SideBar/components/SidebarDropdownMenu";
+import StatusIcon from "../../../../../views/layout/SideBar/components/StatusIcon";
+import styles from "./SideBar.module.scss";
 
 const SideBar: React.FC = () => {
+  const navLinkClassName = useCalculateSidebarStyles();
   const workspace = useCurrentWorkspace();
   const cloudWorkspace = useGetCloudWorkspace(workspace.workspaceId);
+  const config = useConfig();
   const { show } = useIntercom();
   const handleChatUs = () => show();
 
-  const navLinkClassName = useCalculateSidebarStyles();
-
   return (
-    <Bar>
+    <nav className={styles.nav}>
       <div>
         <Link to={workspace.displaySetupWizard ? RoutePaths.Onboarding : RoutePaths.Connections}>
           <img src="/simpleLogo.svg" alt="logo" height={33} width={33} />
         </Link>
         <WorkspacePopout>
-          {({ onOpen, value }) => <WorkspaceButton onClick={onOpen}>{value}</WorkspaceButton>}
+          {({ onOpen, value }) => (
+            <div className={styles.workspaceButton} onClick={onOpen}>
+              {value}
+            </div>
+          )}
         </WorkspacePopout>
-        <Menu>
+        <div className={styles.menu}>
           {workspace.displaySetupWizard ? (
-            <li>
+            <div>
```

Comment:
Is there a reason we switched from a list to `div` here? From semantic HTML perspective the list would make more sense, given that it's a list of links in there.

Reply:
We have the `<li>` tag without the `<ul>` tag. It's not valid markup at all. In this case, I changed it to `<div>`

Reply:
Why not using a `ul` for the wrapping element instead?

Reply:
Yes, we could use a list in a proper way (ul, li), but I don't see real benefits for this.

Reply:
The reason is, that this is the proper semantic HTML for this, which will cause e.g. accessible technologies to understand better what's going on. E.g. a screenreader would read out as soon as the popup opens: "List of 4 items, item 1 out of 4" and will allow the user to get information on which item they are currently focused. In general we should try to strive for clean semantic HTML whenever we can.

Reply:
With next commit, I will change to `ul` `li`